tests/publiccloud/create_aistack_env.pm

```diff
@@ -102,36 +107,58 @@ sub install_aistack_chart {
     my $git_token = get_var('_SECRET_SSH');
     my $ing_ver = get_var('ING_VERSION');
     my $local_storage_name = 'local_path_storage.yaml';
+    my $milvus_custom = 'milvus_custom.yaml';
+    my $milvus_values = 'milvus_values.yaml';
+    my $ollama_values = 'ollama_values.yaml';
+    my $openwebui_values = 'open_webui_values.yaml';
+    my $milvus_helm_repo = 'oci://dp.apps.rancher.io/charts/milvus';
+    my $ollama_helm_repo = 'oci://dp.apps.rancher.io/charts/ollama';
+    my $openwebui_helm_repo = 'oci://dp.apps.rancher.io/charts/open-webui';
 
     record_info('AISTACK charts install');
     assert_script_run("helm list --all-namespaces");
     assert_script_run("kubectl get pods --all-namespaces");
 
     # Access to Application collection registery
-    # Get docker username and password
+    # local-path-storage.yaml is a copy off https://raw.githubusercontent.com/rancher/local-path-provisioner/v0.0.28/deploy/local-path-storage.yaml
     assert_script_run("kubectl create ns $namespace");
     assert_script_run("kubectl create secret docker-registry application-collection --docker-server=dp.apps.rancher.io --docker-username='$docker_user_name' --docker-password='$SECRET_application_collection' -n $namespace", timeout => 120);
+    assert_script_run("curl " . data_url("aistack/$local_storage_name") . " -o $local_storage_name", 60);
+    assert_script_run("kubectl apply -f $local_storage_name", timeout => 120);
+
+    #Install milvus
```

Comment:
Let's have a space between the hash character and the word.

```suggestion
    # Install milvus
```

Reply:
done